main.py

```diff
@@ -152,12 +152,16 @@ def save_passwords():
 
 
 # Function to load passwords from a JSON file 
-def load_passwords():
-     """
-    Load passwords from a file into the password vault.
-
-    This function should load passwords, websites, and usernames from a text
-    file named "vault.txt" (or a more generic name) and populate the respective lists.
+def load_passwords(file_path="vault.json"):
+     try:
+        with open(file_path, 'r') as file:
+            data = json.load(file)
+            return data
```

Comment:
This function is supposed to always return None, so remove this return statement. Instead parse the data to the (global) lists websites, usernames, and encrypted_passwords, like so:
for datum in data:
    websites.append(datum['website'])
    usernames.append(datum['username'])
    encrypted_passwords.append(datum['password_hash'])